app/_src/production/cp-deployment/single-zone.md

```diff
@@ -7,7 +7,7 @@ In order to deploy {{site.mesh_product_name}} in a single-zone deployment, the `
 
 {% tabs usage useUrlFragment=false %}
 {% tab usage Kubernetes %}
-This is the standard installation method. 
+This is the standard installation method. Make sure to restart service pods that are already running after installation such that the data plane proxies can be connected.
```

Comment:
```suggestion
This is the standard installation method. After the installation, make sure to restart application pods that are  running such that the data plane proxies can be generated and connected.
```

Reply:
Here we are assuming the application could have started before the mesh installation. 

I personally feel this is not a practical scenario. Applications need to know whether they need mesh and whether mesh should be enabled on them before they go live. 

That been said, apps that require mesh should not be running before the CP. If they were running for some reason, people should have known they need to restart them.

Reply:
Well, feel free to resolve this conversation. From a test user's perspective, apps and mesh CP can be started in any order. So it's useful for beginers.

Reply:
Yes, I agree that most users know about this 🙂 but as me as a beginner ran into this, I thought it could help.